Fix & Clean karpathy_game notebook

lsqshr opened the pull request:
Since this seems to be the only up-to-date example available now. I believe it would be great to make it work for everyone.
1. `tf.ops.reset_default_graph()` throws exception in released tensorflow. Replaced it with `from tensorflow.python.framework import ops` & `ops.reset_default_graph()` which works fine
2. Removed `from __future__ import print_function`, since it seems not used.
3. Removed the cached results

Pls double check.

lsqshr committed:
Fix the ops in karpathy_game notebook

lsqshr committed:
Fix the ops in karpathy_game notebook

nivwusquorum commented:
Dear Daniel,

Thank you for your pull request, great job! Before I can accept I have a few comments:

1. this is a relic of an old version of TF. Now it is just tf.reset_default_graph()
2. but maybe it was interesting? Don't I do `print(...)` somewhere? If yes now it is printed as tuple...
3. But I want to keep the results in case people just want to see what notebook would have outputed if they run it (github has online notebook viewer).

Let me know if this make sense for you!
----
What word becomes shorter when you add two letters to it?

lsqshr committed:
Fix karpathy game notebook and add results back

1. Change from `ops.reset_default_graph()` to `tf.reset_default_graph()`
1. brought __future__ back
1. made the results
1. Changed `journalist.add_graph(session.graph)` to 1. The last cell
threw exception. I checked the `karpathy_game.py:83`, it should be now
`x[:-4]` rather than `x[:-2]`. Changed.

lsqshr committed:
clean results

lsqshr committed:
clean results

lsqshr committed:
clean results

lsqshr commented:
1. Changed it to `tf.reset_default_graph`
1. brought it back
1. produced the results from another run. Works. Though github seems to have problem with rendering svg images.

additional changes: 
1. changed to `journalist.add_graph(session.graph)` rather than `journalist.add_graph(session.graph_def)` since pass graph_def will be deprecated
1. fixed the last cell by changing `x[:-2]` to x[:-4] according to `karpathy_game.py:83`

---------
The answer is short!

nivwusquorum commented:
Wow, that's awesome thank you for your hard work! You solved the riddle ! ;)
